docs_guide/index.rst

```diff
@@ -1,13 +1,14 @@
-#################################
+=================================
```

Comment:
Is there a reason you prefer using `===` to `###`? It doesn't really matter but I want to unify all files.

Reply:
That's what was used in the [sphinx-guide](https://github.com/Qiskit/qiskit_sphinx_theme/tree/main/docs/sphinx_guide) when I started with the Sphinx-related how-tos. I don't have any strong preference about this.

Reply:
Yeah I notice that too. I don't have strong preference either but since other files are using this, we should stick to it. When we make the templates for the docs guide v2 we can look more into this and decide on one.